app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -283,14 +367,26 @@ class PatcherViewModel(
                             rootInstaller.uninstall(packageName)
                         } catch (_: Exception) {
                         }
+                        isInstalling = false
                     }
                 }
             }
-        } finally {
```

Comment:
There are still cases where `isInstalling` will not have the correct value, such as:
- Successful root install
- [Version conflict early return](https://github.com/ReVanced/revanced-manager/blob/970c3fcd38fbdd595d73e6dd2093820d75b32fa0/app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt#L302-L305)
- [Split APK check early return](https://github.com/ReVanced/revanced-manager/blob/970c3fcd38fbdd595d73e6dd2093820d75b32fa0/app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt#L328-L332)

The easiest way to ensure `isInstalling` is set properly is to put it in a `finally` block like before. The only case where we don't want to unset it is if we have started a regular `PM` install, so create a boolean `var` to track that and check the variable inside the `finally` block.